driver/src/main/java/org/neo4j/driver/Driver.java

```diff
@@ -64,24 +64,24 @@
  */
 public interface Driver extends AutoCloseable {
     /**
-     * Creates a new {@link QueryTask} instance that executes a query in a managed transaction with automatic retries on
+     * Creates a new {@link ExecutableQuery} instance that executes a query in a managed transaction with automatic retries on
      * retryable errors.
      *
      * @param query query string
      * @return new query task instance
-     * @since 5.5
+     * @since 5.6
      */
     @Experimental
-    QueryTask queryTask(String query);
+    ExecutableQuery executableQuery(String query);
 
     /**
-     * Returns an instance of {@link BookmarkManager} used by {@link QueryTask} instances by default.
+     * Returns an instance of {@link BookmarkManager} used by {@link ExecutableQuery} instances by default.
      *
      * @return bookmark manager, must not be {@code null}
      * @since 5.6
      */
     @Experimental
-    BookmarkManager queryTaskBookmarkManager();
+    BookmarkManager defaultExecuteQueryBookmarkManager();
```

Comment:
Do we really want that on the drivers level? 
Simple API should be simple. There's little value here for the users we are aiming for. If we don't need it, we should not introduce it. 
If it is requested, we can still add.

Reply:
As far as I remember, this was introduced to allow access to default  `BookmarkManager` in case someone want to use the same instance in sessions, where an instance must be provided explicitly via the `SessionConfig`.

Reply:
The thinking behind having this method was to allow the user to causally chain with uses of the session API, where a bookmark manager can be supplied on session creation.